controllers/new.go

```diff
@@ -585,6 +633,13 @@ func (k *kubeResources) newGraphBuilderVolumeMounts(instance *cv1.UpdateService)
 			MountPath: "/etc/pki/ca-trust/extracted/pem",
 		})
 	}
+	if k.trustedClusterCAConfig != nil {
```

Comment:
It is not clear to me what happens if both `trustedCAConfig` and `trustedClusterCAConfig` are non-nil, and we have two volume-mounts attempting to compete for the same `/etc/pki/ca-trust/extracted/pem` target.